code/stim/de_VisualizeDataset.m

```diff
@@ -7,19 +7,34 @@
 
   n_pix = prod(dset.nInput);
   img_shape = dset.nInput;
-  im2show     = de_SelectImages(dset, nImages);
+  im2show     = de_SelectImages(dset, nImages); %for vanhateren nImage = 16, but size(im2show) = 20?
 
   fig = de_NewFig('dataset-images', '__img', img_shape, nImages);
   set(fig.handle, 'Position', [0, 0, 1200, 1200]);
 
-  for ii=1:length(im2show)
+  for ii=1:min(4*4, length(im2show)) %make sure only 16 are shown
```

Comment:
Once this constant gets duplicated, good to declare it somewhere and reuse as a constant.